scipy/ndimage/morphology.py

```diff
@@ -1827,45 +1827,49 @@ def distance_transform_bf(input, metric="euclidean", sampling=None,
-        and 'chessboard'.
-    sampling : {int, sequence of ints}, optional
-        This parameter is only used in the case of the euclidean `metric`
-        distance transform.
-
-        The sampling along each axis can be given by the `sampling` parameter
-        which should be a sequence of length equal to the input rank, or a
-        single number in which the `sampling` is assumed to be equal along all
-        axes.
+    metric : {'euclidean', 'taxicab', 'chessboard'}, optional
+        'cityblock' and 'manhattan' are also valid, and map to 'taxicab'.
+        The default is 'euclidean'.
+    sampling : float, or sequence of float, optional
+        This parameter is only used when `metric` is 'euclidean'.
+        Spacing of elements along each dimension. If a sequence, must be of
+        length equal to the input rank; if a single number, this is used for
+        all axes. If not specified, a grid spacing of unity is implied.
     return_distances : bool, optional
-        The `return_distances` flag can be used to indicate if the distance
-        transform is returned.
-
-        The default is True.
+        Whether to calculate the distance transform.
+        Default is True.
     return_indices : bool, optional
-        The `return_indices` flags can be used to indicate if the feature
-        transform is returned.
-
-        The default is False.
-    distances : float64 ndarray, optional
-        Optional output array to hold distances (if `return_distances` is
-        True).
-    indices : int64 ndarray, optional
-        Optional output array to hold indices (if `return_indices` is True).
+        Whether to calculate the feature transform.
+        Default is False.
+    distances : ndarray, optional
+        An output array to store the calculated distance transform, instead of 
+        returning it.
+        This parameter is only used when `return_distances` is True.
+        It must be the same shape as `input`, and of type float64 if `metric` is
+        'euclidean', uint32 otherwise.
+    indices : int32 ndarray, optional
+        An output array to store the calculated feature transform, instead of 
+        returning it.
+        Its shape must be `(input.ndim,) + input.shape`.
 
     Returns
     -------
-    distances : ndarray
-        Distance array if `return_distances` is True.
-    indices : ndarray
-        Indices array if `return_indices` is True.
+    distances : ndarray, optional
+        The calculated distance transform. Returned only when
+        `return_distances` is True and `distances` is not supplied.
+        It will have the same shape as the input array.
+    indices : int32 ndarray, optional
+        The calculated feature transform. It has an input-shaped array for each 
+        dimension of the input. See distance_transform_edt documentation for an 
+        example.
+        Returned only when `return_indices` is True and `indicies` is not 
```

Comment:
Minor nitpick: it should spell `indices` everywhere, not `indicies` :)

Reply:
Thank you!